Sources/ScipioKit/Producer/PIF/XCBuildClient.swift

```diff
@@ -134,21 +134,23 @@ struct XCBuildClient {
         try await executor.execute(arguments)
     }
 
-    private func buildCreateXCFrameworkArguments(sdks: Set<SDK>, debugSymbols: [AbsolutePath]?, outputPath: AbsolutePath) throws -> [String] {
-        let frameworksArguments: [String] = try sdks.reduce([]) { arguments, sdk in
+    private func buildCreateXCFrameworkArguments(sdks: Set<SDK>, debugSymbols: [SDK: [AbsolutePath]]?, outputPath: AbsolutePath) throws -> [String] {
+        let frameworksWithDebugSymbolArguments: [String] = try sdks.reduce([]) { arguments, sdk in
             let path = try frameworkPath(target: buildProduct.target, of: sdk)
-            return arguments + ["-framework", path.pathString]
+            var result = arguments + ["-framework", path.pathString]
+            if let debugSymbols, let paths = debugSymbols[sdk] {
+                paths.forEach { path in
+                    result += ["-debug-symbols", path.pathString]
+                }
+            }
+            return result
         }
```

Comment:
arguments position is important for `-debug-symbols` option.

xcbuild will output error when argument order is like this `-framework <ios-arm64 framework> -framework <ios-arm64_x86_64-simulator framework> -debug-symbols <ios-arm64 dSYM> -debug-symbols <ios-arm64_x86_64-simulator dSYM>`

```
“Alamofire.framework.dSYM” couldn’t be copied to “dSYMs” because an item with the same name already exists.
```